spec/views/layouts/__snapshots__/application_by_election.snap

```diff
@@ -42,7 +42,7 @@
       },
       cookie: {
         name: "_swapmyvote_cookie_consent",
-        domain: "localhost"
+        domain: ""
```

Comment:
There's a fix for this, this part of the snapshot should not be changing. 

You need to create an `env.test.local` .   See https://github.com/swapmyvote/swapmyvote/pull/847/files